sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -557,6 +557,25 @@ class MapboxMap :
     nativeMap.setDebug(debugOptions, enabled)
   }
 
+  /**
+   * Get the Style of the map synchronously, even before style was initially loaded
+   * or when it is transitioning.
+   *
+   * This function should not be used in most cases as accessing and updating Style object
+   * before style loaded and when style is transitioning is not completely safe, please consider
+   * using async [getStyle] instead.
+   *
+   * @return same style as synchronous [getStyle] when Style is loaded and internal Style object otherwise
+   */
+  @MapboxExperimental
+  fun getInternalStyle(): StyleManagerInterface {
```

Comment:
Why not just `nativeMap` independently of `Style` ? 

Also, I'd probably named `getInternalStyleManagerInterface` to make confusion with `Style` less possible